website/docs/r/snat.html.markdown

```diff
@@ -27,7 +26,8 @@ resource "vcd_snat" "outbound" {
 The following arguments are supported:
 
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the SNAT
-* `network_name` - (Optional; *v2.2+*) The name of the organization network on which to apply the SNAT. *`network_name` will be required field in the next major version.*
+* `network_type` - (Optional; *v2.2+*) Type of the network on which to apply the NAT rule. Default: "org". *`network_type` will be required field in the next major version.*
+* `network_name` - (Optional; *v2.2+*) The name of the network on which to apply the SNAT. *`network_name` will be required field in the next major version.*
```

Comment:
"will be required field" -> "will be a required field"

Reply:
changed